.travis/script.d/test.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+# Travis clobbers GOARCH env vars so this is necessary here.
+if [[ -n $FORCEGOARCH ]]; then
```

Comment:
shouldn't this read:
```suggestion
if [[ -n $FORCE_GOARCH ]]; then
```
instead ?

Reply:
Indeed it should. I thought I got all of them.